### PR TITLE
Increase some more timeouts to eliminate flakes

### PR DESCRIPTION
**Pull request by brendandburns**
@quinton-hoole @davidopp @dchen1107 @erictune 

Fixes https://github.com/kubernetes/kubernetes/issues/12312 (I hope)

### CODE REVIEW

**Comment by davidopp** [comment]
This seems fine, though 2 min seems like a long time to wait (assuming the tradeoff is that it makes the test run longer). Did you find any smaller value that also causes the test to pass?

**Comment by brendandburns** [comment]
To be clear, in the expected case, you don't wait 2 minutes.  You poll every second until the condition is true.  So it won't make the test run any longer than it has to.

File: test/integration/scheduler_test.go [hunk 1]
```diff
@@ -253,7 +253,7 @@ func DoTestUnschedulableNodes(t *testing.T, restClient *client.Client, nodeStore
 		}
 
 		// There are no schedulable nodes - the pod shouldn't be scheduled.
-		err = wait.Poll(time.Second, time.Second*10, podScheduled(restClient, myPod.Namespace, myPod.Name))
+		err = wait.Poll(time.Second, time.Second*120, podScheduled(restClient, myPod.Namespace, myPod.Name))
```
Comment (quinton-hoole) [inline test/integration/scheduler_test.go]: Rather use util.ForeverTestTimeout , or a multiple of it if you really must?

File: test/integration/scheduler_test.go [hunk 2]
```diff
@@ -271,7 +271,7 @@ func DoTestUnschedulableNodes(t *testing.T, restClient *client.Client, nodeStore
 		mod.makeSchedulable(t, schedNode, nodeStore, restClient)
 
 		// Wait until the pod is scheduled.
-		err = wait.Poll(time.Second, time.Second*10, podScheduled(restClient, myPod.Namespace, myPod.Name))
+		err = wait.Poll(time.Second, time.Second*120, podScheduled(restClient, myPod.Namespace, myPod.Name))
```
Comment (quinton-hoole) [inline test/integration/scheduler_test.go]: As above.

**Comment by brendandburns** [comment]
@quinton-hoole that's not merged yet.  If you merge it, I will use it ;)

**Comment by quinton-hoole** [comment]
@brendandburns heh heh good point.  #14064 is merged now, so please use it :-)

**Comment by brendandburns** [comment]
@quinton-hoole done.  ptal

thanks!
--brendan

File: test/integration/scheduler_test.go (hunk 1 above)
Comment (brendandburns) [inline test/integration/scheduler_test.go]: done.

File: test/integration/scheduler_test.go (hunk 2 above)
Comment (brendandburns) [inline test/integration/scheduler_test.go]: done.

**Comment by quinton-hoole** [comment]
LGTM.  Why is Shippable failing?

**Comment by davidopp** [comment]
LGTM

**Comment by ixdy** [comment]
Shippable is failing on the integration tests.

One build:
```
 k8s.io/kubernetes/test/integration / integration / TestServiceAccountTokenAuthentication

 : Failed
```

The other:
```
==================
WARNING: DATA RACE
Read by goroutine 254:
  github.com/coreos/go-etcd/etcd.(*Client).getCancelable()
      /root/workspace/src/k8s.io/kubernetes/Godeps/_workspace/src/github.com/coreos/go-etcd/etcd/requests.go:41 +0xcc
  github.com/coreos/go-etcd/etcd.(*Client).get()
      /root/workspace/src/k8s.io/kubernetes/Godeps/_workspace/src/github.com/coreos/go-etcd/etcd/requests.go:62 +0x78
  github.com/coreos/go-etcd/etcd.(*Client).RawGet()
      /root/workspace/src/k8s.io/kubernetes/Godeps/_workspace/src/github.com/coreos/go-etcd/etcd/get.go:31 +0x28e
  github.com/coreos/go-etcd/etcd.(*Client).Get()
      /root/workspace/src/k8s.io/kubernetes/Godeps/_workspace/src/github.com/coreos/go-etcd/etcd/get.go:11 +0x79
      /root/workspace/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/apiserver/resthandler.go:631 +0x140

Previous write by goroutine 98:
  github.com/coreos/go-etcd/etcd.(*Cluster).failure()
      /root/workspace/src/k8s.io/kubernetes/Godeps/_workspace/src/github.com/coreos/go-etcd/etcd/cluster.go:28 +0x69
  github.com/coreos/go-etcd/etcd.(*Client).SendRequest()
      /root/workspace/src/k8s.io/kubernetes/Godeps/_workspace/src/github.com/coreos/go-etcd/etcd/requests.go:264 +0x17a6
  github.com/coreos/go-etcd/etcd.(*Client).getCancelable()
      /root/workspace/src/k8s.io/kubernetes/Godeps/_workspace/src/github.com/coreos/go-etcd/etcd/requests.go:51 +0x4a5
  github.com/coreos/go-etcd/etcd.(*Client).watchOnce()
      /root/workspace/src/k8s.io/kubernetes/Godeps/_workspace/src/github.com/coreos/go-etcd/etcd/watch.go:96 +0x20b
      /root/workspace/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/api/requestcontext.go:101 +0x11c
  net/http.HandlerFunc.ServeHTTP()
      /root/.gvm/gos/go1.3/src/pkg/net/http/server.go:1235 +0x4d
  k8s.io/kubernetes/test/integration.func·001()
      /root/workspace/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/test/integration/utils.go:88 +0x9f
  net/http.HandlerFunc.ServeHTTP()
      /root/.gvm/gos/go1.3/src/pkg/net/http/server.go:1235 +0x4d
  net/http/httptest.(*waitGroupHandler).ServeHTTP()
      /root/.gvm/gos/go1.3/src/pkg/net/http/httptest/server.go:200 +0xf4
  net/http.serverHandler.ServeHTTP()
      /root/.gvm/gos/go1.3/src/pkg/net/http/server.go:1673 +0x1fc
  net/http.(*conn).serve()
      /root/.gvm/gos/go1.3/src/pkg/net/http/server.go:1174 +0xf9e
==================
```

**Comment by jszczepkowski** [comment]
@ixdy These failures are not related to this PR.
Shippable is passing now, merging.

**Pull request merged**